src/mixins/state.js

```diff
@@ -173,6 +173,21 @@ export default {
     return this._stateModel.get.apply(this._stateModel, arguments);
   },
 
+  /**
+   * Toggle a property on the _stateModel.
+   *
+   * @public
+   * @method toggleState
+   * @param {String} attr - Attribute name of stateModel.
+   * @param {val} [value] - Attribute value if key is String.
+   * @returns {Backbone.Model} - The _stateModel
```

Comment:
the return value will be the state model or the value set I believe.  same as `setState`